po/pt-BR.po

```diff
@@ -6281,11 +6627,11 @@ msgid ""
 "    }\n"
 "}"
 msgstr ""
-"fn divide_in_two(n: i32) -> Result {\n"
+"fn divide_por_dois(n: i32) -> Resultado {\n"
 "    if n % 2 == 0 {\n"
-"        Result::Ok(n/2)\n"
```

Comment:
It's great to translate the strings and comments, thank you for all this. One thing we shouldn't translate is `Result`, since this is a [type](https://doc.rust-lang.org/std/result/), please would you change the translated 'Resultado's back to 'Result's.